moveit_ros/visualization/planning_scene_rviz_plugin/include/moveit/planning_scene_rviz_plugin/planning_scene_display.h

```diff
@@ -36,44 +36,29 @@
 
 #pragma once
 
-#include <rviz/display.h>
-
+#include <rviz_common/display.hpp>
+#include <rviz_default_plugins/robot/robot.hpp>
+#include <rviz_common/properties/string_property.hpp>
+#include <rviz_common/properties/ros_topic_property.hpp>
 #ifndef Q_MOC_RUN
 #include <moveit/rviz_plugin_render_tools/planning_scene_render.h>
 #include <moveit/planning_scene_monitor/planning_scene_monitor.h>
 #include <moveit/background_processing/background_processing.h>
-#include <ros/ros.h>
+#include <rclcpp/rclcpp.hpp>
 #endif
 
-namespace Ogre
```

Comment:
Aren't these used anymore?

Reply:
They're just used for the forward declaration of these classes, I don't think we need them anymore